### DISCUSSION TITLE
Resistance-power curve

### DISCUSSION THREAD

**Discussion by HilmiiKumdakci** [cagnulein/qdomyos-zwift#297] [Q&A]
Hi,

As a practitioner in ML, I would like to see and  put some work on the power-cadence-resistance file for Domyos Bike 500 if you have and if you would like to share or if it exists in the repository.

Thanks.

**Comment by cagnulein** [comment]
Yes it's the src/domyosbike.cpp

Il sab 1 mag 2021, 19:05 HilmiiKumdakci ***@***.***> ha
scritto:
